### ISSUE TITLE
[FEATURE REQUEST] Add button to go from docs.novu.co to novu.co

### ISSUE THREAD

**Issue by PritamSarbajna** [novuhq/website#20]
### describe the issue :

In _docs.novu.co_ there isn't anything which lead user from `docs.novu.co` to `novu.co`

### suggestive solution:

-  We can add a button at navbar to `novu.co`
or
-  adding `novu.co` to the novu logo also solves the problem
or
-  We can add something on footer which will redirect user to `novu.co`

**Comment by andrewgolovanov** [comment]
Hi @PritamSarbajna!
Thank you so much for the feedback 🙌
This is very helpful!

I like the solution of adding a link to the logo, we can also add the `target` attribute with the value `_blank` which allows you to simply open `novu.co` in a new tab without leaving the documentation page

@scopsy Please tell me what you think about it?

**Comment by PritamSarbajna** [comment]
Yeah @andrewgolovanov that'd be great. can you tell me where I can implement this as a button or something or like _checkout novu.co_ so I can contribute to it

**Comment by ComBarnea** [comment]
Hey @andrewgolovanov @PritamSarbajna I love this idea!

@PritamSarbajna as to your question, docs are managed [here](https://github.com/novuhq/novu/tree/main/docs), so on another repo (novuhq/novu), I am moving this issue there. As to the question of the logo link change, take a look into docusaurus docs about this [here](https://docusaurus.io/docs/api/themes/configuration).

@PritamSarbajna should we assigned this to you?
